src/Components/NewTodo.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { Fragment, useState } from "react";
+import { useDispatch } from "react-redux";
+import { addTodo } from "services/actions/addTodo";
+
+export default function NewTodo(props) {
```

Comment:
when passing props, its good practice to specify the props that is needed instead of a generic prop-data. Also you can use propTypes package to handle prop data from getting unexpected error. Thanks!

Reply:
addressed